pipelines/unittests/Jenkinsfile

```diff
@@ -1,5 +1,31 @@
-pipeline {
+// This function calculates the Java version from JAVA_HOME
+def calculateJavaVersion() {
+    def javaVersion = sh(script: "echo ${JAVA_HOME} | awk -F'/' '{print \$NF}'", returnStdout: true).trim()
+    return javaVersion
+}
+
+// This function calculates the Tomcat folder from the URL
+def calculateTomcatFolder() {
+    def tomcatFolder = sh(script: "basename ${TOMCAT_URL} .tar.gz", returnStdout: true).trim()
+    return tomcatFolder
+}
+
+// This function generates the stack message using calculated values
```

Comment:
Totally personal opinion, but "_this function does X_" will not be better to rewrite it as "_Does X_" directly?

Reply:
Done! Thanks @Gremiger 😄